src/main/duke/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.TodoTask;
+
+public class Storage {
+
+    private File file;
+    
+    public Storage(String filePath) {
```

Comment:
Don't forget to write header comments for all classes. :)

Reply:
Thanks for the reminder. I'll add the comments soon :)